subscriber/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: subscriber-secret
+  namespace: {{ .Release.Namespace }}
+data:
+  INFRA_ID: "{{ .Values.global.infraId }}"
```

Comment:
Can we add required check to all the required fields 
https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function